app/views/assets/view.php

```diff
@@ -118,6 +118,11 @@
                         <div class='panel-heading' style='font-size: large; padding-bottom: 15px'>
                             <b>Asset Name: <?php echo $assets['name']; ?> </b>
                             <div class='btn btn-info show' style='float: right;'>Show More</div>
+
+                            <a href="update?id=<?php echo $assets['id'] ?>">
+
+                                <div class='btn btn-success update' style='float: right;'>Update</div>
```

Comment:
Add a small amount of margin to the right of this button, i.e, **margin-right: 2px;**